insights/terraform/projects/bb2/README.md

```diff
@@ -12,43 +12,65 @@ The BB2 specific components are setup under the following service areas:
+  - Is updated when adding new metrics.
 
-## Usage
+## Terraform Usage
 
 To initially work with in a service area, use the following commands. For this example, the `log_stream` service is used:
 
 ```
 cd services/log_stream
-tfenv install 1.5.0 # install tf version needed
-tfenv use 1.5.0 # select tf version to use
+tfenv install 1.6.5 # install tf version needed
+tfenv use 1.6.5 # select tf version to use
 terraform init # locally initialize 
 ```
 
-There are `terraform.tfvars` files that contain sensitive variables. They are located under the BB2 team S3 storage.  In this example, the related file is located under this path:  `infrastructure/terraform_vars/bfd-insights/bb2/services/log_stream/terraform.tfvars`.
+There are `terraform.tfvars` files that contain sensitive variables. They are located in BB2 CMS Box.
 
-Copy this file in to the current working directory.
+Copy the file related to the target service in to the current working directory, if the service requires one.
 
-The Terraform workspaces feature is used for switching between `test/impl/prod` environments.
+The Terraform workspaces feature is used for switching between `test/impl/prod/allenv` environments.
 
-The following is an example of working with in the `test` environment:
+The following is an example of working with in the `test` environment for this service:
 
 ```
-terraform workspace list # to see the list of envs available
-terraform workspace select test # to switch to test env
+terraform workspace list # to see the list of workspaces available
+terraform workspace select test # to switch to test workspace
 
 terraform plan # to see the plan for changes
 terraform apply # to apply changes after review
+
+terraform plan # Review the changes still showing after the APPLY.
 ```
+
+**WARNING**: The plans **must only** show resources to be changed!  If you are seeing any to DELETE or CREATE, something is wrong. For example, being in the default workspace will show all resources to be created.
+
+## Special Note About Quicksight Analyses 
+
+Normally when terraforming resources, the `terraform plan` after a `terraform apply`, will show the following message: "No changes. Your infrastructure matches the configuration."
+
+However, due to several issues when terraforming Quicksight Analyses, these will always show some types of expected changes. These are described in the following README document: [services/quicksight/README.md](services/quicksight/README.md).
+
+
+## Documentation for Athena & Quicksight Metrics
+
+There is more detailed documention located under the `update_athena_metric_tables` lambda module directory. This contains documentation related to updating the intermediate tables utilized by the datasets. The datasets are used by the analyses that are used to publish the dashboards.
+
+For more details, see this [services/analytics/modules/lambda/update_athena_metric_tables/README.md](services/analytics/modules/lambda/update_athena_metric_tables/README.md).
```

Comment:
@dtisza1 I think it would be a good idea to add my finding of the bug with adding boolean metrics inside the readme under update_athena_metric_tables. I can do that if you'd like!

Reply:
> @dtisza1 I think it would be a good idea to add my finding of the bug with adding boolean metrics inside the readme under update_athena_metric_tables. I can do that if you'd like!

I'm going to work on adding to that README next. I'll include something about that. Probably not good to update the code to handle boolean values now. Not wanting to break anything :-)    Might be good to put in a backlog Jira story about it?

Reply:
@dtisza1  I agree! I can open a ticket on that bug

Reply:
@meliGuzman TY! That would work out great with a backlog story for that bug.

Reply:
@dtisza1 bug ticket created! https://jira.cms.gov/browse/BB2-3133